pkg/apis/flowcontrol/validation/validation_test.go

```diff
@@ -1375,7 +1340,7 @@ func TestValidateLimitedPriorityLevelConfiguration(t *testing.T) {
 			}
 			specPath := field.NewPath("spec").Child("limited")
 
-			errGot := ValidateLimitedPriorityLevelConfiguration(configuration, test.requestVersion, specPath, PriorityLevelValidationOptions{AllowZeroLimitedNominalConcurrencyShares: test.allowZero})
+			errGot := ValidateLimitedPriorityLevelConfiguration(configuration, test.requestVersion, specPath, PriorityLevelValidationOptions{})
```

Comment:
is removing these correct? I thought we now allow zero and all these test must run unconditionally?

Reply:
@sttts It is right to remove the field because its value is always true and I also change the validation. Please see https://github.com/kubernetes/kubernetes/pull/126894/files#diff-8f99dc060b375b8082f4959db496ce7114167b565274f99bcc66900d1245dd1eR427

Reply:
- https://github.com/kubernetes/kubernetes/pull/126894/files#diff-e652a3bbbc9fb3c345ef579829b78977cbd17f46b9caee8aa288d303a73abaa7L103
- https://github.com/kubernetes/kubernetes/pull/126894/files#diff-e652a3bbbc9fb3c345ef579829b78977cbd17f46b9caee8aa288d303a73abaa7L144